environment.yml

```diff
@@ -1,8 +1,7 @@
 name: lst-dev
 channels:
-  - default
-  - cta-observatory
   - conda-forge
+  - default
```

Comment:
why changing the order?

Reply:
To give priority to the conda-forge packages, which tend to be more up to date and are actually open source